### ISSUE TITLE
[Concepts] Slicing the monolith

### ISSUE THREAD

**Issue by ignasi35** [akka/akka-platform-guide#245]
Add a section in concepts on how to slice the monolith.

**Comment by patriknw** [comment]
That seems to be an incredibly huge and "it depends" topic. Shall we really try to cover that?

**Comment by patriknw** [comment]
also, seems to be an exact duplicate of https://github.com/akka/akka-platform-guide/issues/170

**Comment by ignasi35** [comment]
This topic is not a stopper and I agree that it can grow tremendously. 

I'd like to add something minimal, just a few paragraphs (not more than four) with the big strokes of the general steps required to Slice the Monolith. Here's some ideas 

 - separate bounded contexts (link to ddd.adoc): separate DB tables VS related DB tables 
 - strong consistent boundaries vs eventual consistent boundaries
 - emit events on state change - data pumps (the service is the owner of the events and those are API)

(maybe link to the chapter on sam newman's book)

Nothing more.

**Comment by octonato** [comment]
I don't think we should add anything on those lines. Otherwise, the guide will become a online book on software architecture. I think we should only describe the concepts that are immediately related to our tech stack and needed to make better use of it.

The transition from Monolith to Microservices is not directly related to Akka.
